File: Ruby/example_usage.rb
# example_usage.rb
# Example usage of the BinomialCoefficient class

require_relative 'binomial_coefficient'

# Parameters for binomial coefficient
n = 5
k = 2

# Calculate the binomial coefficient
result = BinomialCoefficient.calculate(n, k)

# Display the result
puts "Binomial coefficient C(#{n}, #{k}) is: #{result}"

File: C/example_usage.c
// example_usage.c
// Example usage of the binomial_coefficient function

#include <stdio.h>
#include "binomial_coefficient.h"

int main() {
    int n = 5;
    int k = 2;

    int result = binomial_coefficient(n, k);

    printf("Binomial coefficient C(%d, %d) is: %d\n", n, k, result);

    return 0;
}
